src/SourceCompile/AnalyzeFile.cpp

```diff
@@ -104,7 +104,11 @@ void AnalyzeFile::analyze() {
     std::string line;
     std::stringstream ss(m_text);
     while (std::getline(ss, line)) {
-      allLines.push_back(line);
```

Comment:
Please remove this change from the PR.
It has nothing do to with the rest of the PR. Please explain the rational here.